.doc_gen/metadata/dynamodb_metadata.yaml

```diff
@@ -240,8 +272,17 @@ dynamodb_DeleteItem:
             - description:
               snippet_tags:
                 - dynamodb.python.codeexample.MoviesItemOps06
+    Rust:
+      versions:
+        - sdk_version: 1
+          github: .rust_alpha/dynamodb
+          excerpts:
+            - description:
+              snippet_tags:
+                - dynamodb.rust.crud-remove_item
+                - dynamodb.rust.delete-item

```

Comment:
These are so similar, I'd pick one and remove the other.

Reply:
I've removed the duplicates.